base/process/memory_linux.cc

```diff
@@ -111,7 +111,10 @@ bool UncheckedMalloc(size_t size, void** result) {
 #elif defined(MEMORY_TOOL_REPLACES_ALLOCATOR) || !defined(LIBC_GLIBC)
   *result = malloc(size);
 #elif defined(LIBC_GLIBC)
+// TODO(cobalt, b/382115610): Re-enable use_partition_alloc
+#if !BUILDFLAG(IS_STARBOARD)
```

Comment:
This isn't what we want to do—if we use this file in c25, do what we do there, but I assume we use memory_starboard.cc instead. If that's the case, then we likely want to ensure either `MEMORY_TOOL_REPLACES_ALLOCATOR` is defined or `LIBC_GLIBC` isn't—I'm not sure which is more appropriate.

Reply:
thanks, ill try adding memory_starboard.cc

Reply:
done, added